### ISSUE TITLE
Danish translations are outdated

### ISSUE THREAD

**Issue by brammool** [vim/vim#10161]
### Steps to reproduce

Start Vim with language set to Danish.
Observe that many messages are in English

### Expected behaviour

Most messages are translated to Danish.

### Version of Vim

8.2

### Environment

Danish

**Comment by brammool** [comment]
@scootergrisen

**Comment by scootergrisen** [comment]
I think it will be better to work on making the translations easier to translate.
I have suggested this before.
Vim is very hard to translate/keep fully translated compared to other software.

**Comment by brammool** [comment]
> I think it will be better to work on making the translations easier to
> translate.
> I have suggested this before.
> Vim is very hard to translate/keep fully translated compared to other
> software.

Can you add a pointer to these suggestions?

-- 
Scientists decoded the first message from an alien civilization:
        SIMPLY SEND 6 TIMES 10 TO THE 50 ATOMS OF HYDROGEN TO THE STAR
SYSTEM AT THE TOP OF THE LIST, CROSS OFF THAT STAR SYSTEM, THEN PUT
YOUR STAR SYSTEM AT THE BOTTOM OF THE LIST AND SEND IT TO 100 OTHER
STAR SYSTEMS.  WITHIN ONE TENTH GALACTIC ROTATION YOU WILL RECEIVE
ENOUGH HYDROGREN TO POWER YOUR CIVILIZATION UNTIL ENTROPY REACHES ITS
MAXIMUM!  IT REALLY WORKS!

 /// Bram Moolenaar -- ***@***.*** -- http://www.Moolenaar.net   \\\
///                                                                      \\\
\\\        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by chrisbra** [comment]
I suppose that is: https://github.com/vim/vim/issues/2568 and we had this part inside todo.txt for a while:

https://github.com/vim/vim/blob/e46a4405056276b4cbdacee76b11f85c8ea1830b/runtime/doc/todo.txt#L41

I think this was part for a discussion we had some time ago on making translations more easily available. I somehow tested it using [gitlocalize](https://gitlocalize.com/)  some time ago and I think to actually use this, we would need the check in the source strings in the po files (similar to what en_GB.po has now I think). Perhaps one could see it here: https://gitlocalize.com/repo/7330
But do not remember many details unfortunately.

**Comment by brammool** [comment]
> I suppose that is: https://github.com/vim/vim/issues/2568 and we had this part inside todo.txt for a while:
> 
> https://github.com/vim/vim/blob/e46a4405056276b4cbdacee76b11f85c8ea1830b/runtime/doc/todo.txt#L41
> 
> I think this was part for a discussion we had some time ago on making translations more easily available. I somehow tested it using [gitlocalize](https://gitlocalize.com/)  some time ago and I think to actually use this, we would need the check in the source strings in the po files (similar to what en_GB.po has now I think). Perhaps one could see it here: https://gitlocalize.com/repo/7330
> But do not remember many details unfortunately.

The discussion comes down to not willing to follow the steps to create
translations.  They want to do one step and have others do the rest.
Generating the .pot file is easy enough, but it also includes running
the check and clean scripts, to make sure the translations have been
done properly (e.g. printf format strings did not got mangled).  The
main pain point with this is that if the translater produces rubbish,
someone else has to check and report back problems.

I expect someone who understands Vim to do the translations, and getting
Vim from github and running "make" is so simple that if someone doesn't
even want to make these simple steps I very much doubt the translations
will be any good.  I mean, if you don't get the latest Vim, how are you
going to check the translations in context?  Aren't you going to use Vim
with your own translations?  Might as well have Google translate do the
work (which is likely crappy).

-- 
10e12 microphone == 1 megaphone

 /// Bram Moolenaar -- ***@***.*** -- http://www.Moolenaar.net   \\\
///                                                                      \\\
\\\        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by chrisbra** [comment]
yes indeed. Personally I find the work not to be very hard. But then I am used to the development process in Vim. 

However for the user that notices a wrong translation and just want to contribute this quickly back, using the mentioned gitlocalize might make sense, since this may quite lower the burden.

**Comment by scootergrisen** [comment]
Just keep it the way it is then if you think its best.
